syntax="proto3";
option cc_generic_services = true;
option java_generic_services = true;

package drpc;

message SubscribeRequest {
    string message = 1;
};

message SubscribeResponse {
    string message = 1;
};

service PubSubService {
    rpc Request(SubscribeRequest) returns (SubscribeResponse);
}
